src/main/scala/edu/berkeley/ce/rockslicing/InputProcessor.scala

```diff
@@ -79,45 +72,31 @@ object InputProcessor {
         return None
       }
       val doubleVals = tokens map(_.get)
-      if (doubleVals.length < 11) {
-        println(s"""Error, Line $index: Each input joint is defined by at least 11 values:
-                        3 For Normal Vector
-                        3 For Local Origin
-                        3 For Joint Center
+      if (doubleVals.length < 6) {
+        println(s"""Error, Line $index: Each input joint is defined by at least 6 values:
+                        Strike
+                        Dip
+                        Joint Spacing
+                        Persistence (Input as percentage, 100 for persistent joints)
                         Phi
                         Cohesion
-                        4 For Each Bounding Face (Optional)
+                        Optional input: Standard deviation of 4 geometric parameters. Standard deviation
+                        should not be specified for persistence when set to 100%
                 """
         )
         return None
       }
 
-      val (mandatoryValues, optionalValues) = doubleVals.splitAt(11)
-      if (optionalValues.length % 4 != 0) {
-        println(s"Error, Line $index: Bounding Faces for Block Require 4 Values Each")
+      val optionalValues = doubleVals.drop(6)
```

Comment:
I think you could simplify this by getting rid of the `optionalValues` val and then checking the length of `doubleVals` against 6, 9, and 10 rather than checking the length of `optionalValues` against 0, 3 and 4.